include/oxen/quic/utils.hpp

```diff
@@ -112,6 +112,8 @@ namespace oxen::quic
     inline constexpr std::chrono::seconds DEFAULT_HANDSHAKE_TIMEOUT = 10s;
     inline constexpr std::chrono::seconds DEFAULT_IDLE_TIMEOUT = 30s;
 
+    inline constexpr size_t inverse_golden_ratio = sizeof(size_t) >= 8 ? 0x9e37'79b9'7f4a'7c15 : 0x9e37'79b9;
```

Comment:
```suggestion
    // quasi-random fixed value used in multi-value hash computations
    inline constexpr size_t inverse_golden_ratio = sizeof(size_t) >= 8 ? 0x9e37'79b9'7f4a'7c15 : 0x9e37'79b9;
```
Previously it was defined just above where being used, but now it's separated quite a bit so deserves a comment.

Reply:
Looking again: I don't understand why this got moved into `utils.hpp` when it's only an implementation detail of `address.hpp`.

Reply:
Mentioned this off-band but putting here: bringing this up into application code, similar to the `to_string_formattable` concept